### PR TITLE
NAKIJIN-602. Change accommodationId parameters to htAccommodationId

### PR DESCRIPTION
**Pull request by oleg1std**
Change accommodationId parameters to htAccommodationId in the endpoints of AccommodationManagementController

### CODE REVIEW

File: HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs [hunk 1]
```diff
@@ -562,16 +396,252 @@ Task Unpublish(List<RichAccommodationDetails> accommodations)
...
+            if (isFailure)
+                return Result.Failure<int>(error);
+            
+            if (type != MapperLocationTypes.Accommodation)
+                return Result.Failure<int>($"htAccommodationId '{htAccommodationId}' has an invalid type '{type.ToString()}'. The type must be equal '{MapperLocationTypes.Accommodation.ToString()}'");
+
+            return id;
+        }
+        
+
```
Comment (kirillta) [inline HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs:644]: Could you split this file? 650 lines is too big to track its behavior

Comment (oleg1std) [reply HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs:644]: yes I could

**Review by mingazhev — APPROVED** [review]

**Review by AlexeyShok — APPROVED** [review]

File: HappyTravel.Nakijin.Api/Models/Accommodations/RemoveSupplierRequest.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace HappyTravel.Nakijin.Api.Models.Accommodations
+{
+    public readonly struct RemoveSupplierRequest
+    {
+        [Required]
+        public Suppliers Supplier { get; init; }
+        
+        [Required]
+        public string SupplierId { get; init; }
```
Comment (mingazhev) [inline HappyTravel.Nakijin.Api/Models/Accommodations/RemoveSupplierRequest.cs:12]: SupplierAccommodationId?
or maybe renaming model to `RemoveSupplierAccommodationRequest` or something similar?
Now it looks like we delete the whole supplier from somewhere

**Review by mingazhev — APPROVED** [review]
With a small comment

**Review by kirillta — APPROVED** [review]

File: HappyTravel.Nakijin.Api/Models/Accommodations/RemoveSupplierRequest.cs (hunk 2 above)
Comment (oleg1std) [reply HappyTravel.Nakijin.Api/Models/Accommodations/RemoveSupplierRequest.cs:12]: namespace is HappyTravel.Nakijin.Api.Models.Accommodations

Comment (oleg1std) [reply HappyTravel.Nakijin.Api/Models/Accommodations/RemoveSupplierRequest.cs:12]: fixed

**Pull request merged**